bionic/datatypes.py

```diff
@@ -295,13 +298,30 @@ def str_from_version_value(value):
 class CodeVersion:
     """
     Contains the user-designated version of a piece of code, consisting of a
-    major and a minor version string.  The convention is that changing the
-    major version indicates a functional change, while changing the minor
-    version indicates a nonfunctional change.
+    major and a minor version string, and a boolean that indicates whether it
+    includes the bytecode.  The convention is that changing the major version
+    indicates a functional change, while changing the minor version indicates a
+    nonfunctional change.  includes_bytecode determines whether the bytecode is
```

Comment:
I would phrase the last sentence as, "If `include_bytecode` is True, then the major version is understood to implicitly include the bytecode of the code as well."